verticapy/learn/ensemble.py

```diff
@@ -416,6 +416,54 @@ def get_prior(self):
         return prior_
 
 
+# ---#
+class IsolationForest(Clustering, Tree):
+    """
+---------------------------------------------------------------------------
+Creates an IsolationForest object using the Vertica IFOREST algorithm.
+
+Parameters
+----------
+name: str
+    Name of the the model. The model is stored in the DB.
+n_estimators: int, optional
+    The number of trees in the forest, an integer between 0 and 1000, inclusive.
+max_depth: int, optional
+    Maximum depth of each tree.
+nbins: int, optional
```

Comment:
I wonder if you want to mention about the valid range of max_depth and nbins as well.

Reply:
Done;